app/applicant/jobs/page.tsx

```
@@ -4,29 +4,44 @@
+  id: string;
+  title: string;
+  company: string;
+  location: string;
+  type: string;
+  salary: string;
+  description: string;
+  requirements: string[];
+  postedDate: string;
+}
+
 interface JobFilters {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'JobFilters'. Consider to delete it or rename it.

'JobFilters' is defined here:


(lint/suspicious/noRedeclare)